src/AppBundle/Resources/views/Report/ProfDeputyCostsEstimate/_answers.html.twig

```diff
@@ -37,7 +37,7 @@
                     {{ 'breakdown.form.profDeputyCostsEstimateManagementCost.label' | trans }}
                 </td>
                 <td>
-                    £{{ (report.profDeputyManagementCostAmount) | money_format }}
+                    £{{ (report.getProfDeputyManagementCostAmount) | money_format }}
```

Comment:
Just for info, you don't need to specify 'get', trig automatically calls a get method based on the property you want

Reply:
Ah nice, wasn't aware of that. I think I added this during debugging why it wasn't pulling through until I realised the property name wasn't matching from the api. I'll switch back to `profDeputyManagementCostAmount ` for consistency 👍